accessing
internalErrors: anObject
	internalErrors := anObject